Makefile.am

```diff
@@ -8,6 +8,7 @@ bin_PROGRAMS = \
 	simple/fi_rdm \
 	simple/fi_rdm_rma_simple \
 	simple/fi_rdm_rma_trigger \
+	simple/fi_rdm_triggeredOps \
```

Comment:
Did you really introduce CamelCase?  :)  Based on a quick inspection, it looks like the test uses the deferred_wq concept, so I would rename to reflect that.